closed/custom/modules/java.base/Copy.gmk

```diff
@@ -30,7 +30,7 @@ $(call openj9_copy_files,, \
 	$(OPENJ9_VM_BUILD_DIR)/include/jvmti.h \
 	$(INCLUDE_TARGET_DIR)/jvmti.h)
 
-ifeq (zos,$(OPENJDK_TARGET_OS))
```

Comment:
There are three other files that use this pattern: could you fix them as well?

Reply:
which files?

Reply:
```
closed/OpenJ9.gmk
closed/make/modules/ibm.healthcenter/Copy.gmk
```